test/hclwrite/hclwrite.go

```diff
@@ -222,7 +222,11 @@ func NumberInt(name string, val int64) BlockBuilder {
 
 // Format formats the given HCL code.
 func Format(code string) string {
-	return strings.Trim(string(hclwrite.Format([]byte(code))), "\n ")
+	formatted, err := hcl.Format(code, "gen.hcl")
+	if err != nil {
```

Comment:
One alternative to assuming that only valid HCL is allowed when using the hclwrite builders would be to just skip formatting if the code is not valid HCL (or maybe falling back to hclwrite.Format that formats anything). But the alternative seems clumsy, overall I think we want to build valid HCL using the builders, so if because a bug it is not valid HCL it is desireable for the tests to fail fast.